### PR TITLE
Handle missing answers in aggregation differently - do a little intel…

### PR DESCRIPTION
**Pull request by mehalshah**
…ligent searching, and then proceed and notify HB

### CODE REVIEW

File: dashboard/app/helpers/pd/workshop_survey_results_helper.rb [hunk 1]
```diff
@@ -431,13 +433,46 @@ def generate_facilitator_averages(summary)
...
-          question[:option_map][k] * v
+          option = question[:option_map][k]
+
+          if option.nil?
+            # special case for cases where in the scale we changed the value
+            option = question[:option_map].select {|k1, _| k1.start_with? k.to_s}.values.first
+          end
+
+          if option
+            option * v
```
Comment (clareconstantine) [inline dashboard/app/helpers/pd/workshop_survey_results_helper.rb:444]: what does this do? google wasn't particularly helpful.

**Review by clareconstantine — APPROVED** [review]
Do we need a test for this?

File: dashboard/app/helpers/pd/workshop_survey_results_helper.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/app/helpers/pd/workshop_survey_results_helper.rb:444]: I can better document it.

option is the numeric value that we assign to a possible selection. The above code builds maps for each question that will turn an option list of `[poor, fair, good, very good, excellent]` to `{poor => 1, fair => 2, good => 3, very good => 4, excellent => 5}`. Then we multiply that value by the number of times it occurs to get the sum that will be averaged later.

Comment (clareconstantine) [reply dashboard/app/helpers/pd/workshop_survey_results_helper.rb:444]: oh! that makes sense. I thought this was the splat operator or something. thanks!

**Pull request merged**